internal/api/genqlient.graphql

```diff
@@ -113,3 +113,39 @@ query containerRepository($orgId: ID!, $artifactId: ID!, $input: ContainerReposi
     token, repoUri
   }
 }
+
+query getAllArtifacts($organizationId: ID!) {
+  artifacts(organizationId: $organizationId, input: {}) {
+    next
+    items {
+      id
+      name
+      type
+      data
+      artifactDefinition {
+        id
+        name
+        schema
+        url
+      }
+    }
+  }
+}
+
+query getAllArtifactsPaginated($organizationId: ID!, $after: String!) {
+  artifacts(organizationId: $organizationId, input: {after: $after}) {
+    next
+    items {
+      id
+      name
+      type
+      data
+      artifactDefinition {
+        id
+        name
+        schema
+        url
+      }
+    }
+  }
+}
```

Comment:
Awkward and I hate it, but I can't pass a `nil` string? Maybe I'm just not used to Go's type system...

Reply:
Not sure I follow. Is the backend typed not nil? I see a “!” In the Gql

you’ll need a string pointer to pass nil or pass a “nilish” string “”